modules/post/multi/gather/firefox_creds.rb

```diff
@@ -329,31 +407,36 @@ def get_ff_and_loot_path
 
     @paths['ff'] = check_paths.find do |p|
       check = p.sub(/(\\|\/)(mozilla\s)?firefox.*/i, '')
-      print_status("Checking for Firefox directory in: #{check}")
+      vprint_status("Checking for Firefox directory in: #{check}")
       if directory?(p.sub(/(\\|\/)$/, ''))
-        print_good("Found Firefox directory")
+        print_good("Found Firefox directory: #{check}")
         true
       else
-        print_error("No Firefox directory found")
+        #vprint_error("No Firefox directory found")
         false
       end
     end
 
-    return false if @paths['ff'].nil?
+    if @paths['ff'].nil?
+      print_error("No Firefox directory found")
+      return false
+    end
 
     @paths['loot'] = tmpdir + loot_file
     return true
-
   end
 
-  def modify_omnija(zip)
+
+  def decrypt_modify_omnija(zip)
+    # Which files to extract from ja/zip
     files = [
-                'components/storage-mozStorage.js',
-                'chrome/toolkit/content/passwordmgr/passwordManager.xul',
-                'chrome/toolkit/content/global/commonDialog.xul',
-                'jsloader/resource/gre/components/storage-mozStorage.js'
+              'components/storage-mozStorage.js',                        # stor_js
+              'chrome/toolkit/content/passwordmgr/passwordManager.xul',  # pwd_xul
+              'chrome/toolkit/content/global/commonDialog.xul',          # dlog_xul
+              'jsloader/resource/gre/components/storage-mozStorage.js'   # res_js (not 100% sure why this is used)
```

Comment:
Don't know why it's not there for you - but it's required.
Added a basic check/test.